samtools/README.md

```diff
@@ -6,7 +6,7 @@ git clone https://github.com/googlegenomics/pipelines-api-examples.git
 cd pipelines-api-examples/samtools/
 docker build -t ${USER}/samtools .
 ```
-## (2) Test the pipeline locally.
+## (2) Test the Docker image for the pipeline locally.
```

Comment:
"Some of the words have got altered," said Alice :)  Maybe one of these would speak more directly:

> Locally test the Docker image used by the pipeline.

or

> Test locally the Docker image used by the pipeline.

or

> Test-Run locally the Docker image used by the pipeline.